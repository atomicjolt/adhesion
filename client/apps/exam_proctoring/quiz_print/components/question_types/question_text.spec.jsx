import React from 'react';
import { shallow } from 'enzyme';
import TextOnly from './question_text';

describe('text Only', () => {
  let result;
  let props;

  beforeEach(() => {
    props = {
      question_text: '<p>I am a text</p>',
      id: '7',
    };
    result = shallow(<TextOnly {...props} />);
  });

  it('matches the snapshot', () => {
    expect(result).toMatchSnapshot();
  });
});
