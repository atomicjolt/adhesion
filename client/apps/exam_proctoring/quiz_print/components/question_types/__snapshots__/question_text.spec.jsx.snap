// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text Only matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <essayQuestion
    id="7"
    question_text="<p>I am a text</p>"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "dangerouslySetInnerHTML": Object {
        "__html": "<p>I am a text</p>",
      },
    },
    "ref": null,
    "rendered": null,
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "dangerouslySetInnerHTML": Object {
          "__html": "<p>I am a text</p>",
        },
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
